### DISCUSSION TITLE
Welcome to govuk-design-system-discussions-prototype Discussions!

### DISCUSSION THREAD

**Discussion by timpaul** [alphagov/govuk-design-system-discussions-prototype#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timpaul** [comment]
Test comment

**Reply by timpaul** [reply]
Testing threads

**Reply by timpaul** [reply]
More testing of threads!

**Comment by timpaul** [comment]
TEST

**Reply by timpaul** [reply]
REPLY

**Reply by 36degrees** [reply]
HI TIM!

**Reply by CharlotteDowns** [reply]
## Markup test
- [ ] OOOooo we can add all the markup

and link to those other discussions #3
